ksql-engine/src/test/resources/query-validation-tests.json

```diff
@@ -0,0 +1,340 @@
+{
+  "tests": [
```

Comment:
Probably would be good to add a comment that you can have multiple statements per test case, but the last is assumed to be a query and is the only one whose topology is verified.